src/gui/properties/propertieswidget.ui

```diff
@@ -142,7 +142,7 @@
             </widget>
            </item>
            <item>
-            <widget class="Line" name="line_2">
+            <widget class="Line" name="line">
```

Comment:
If it's used from our c++ code, please rename it to some more meaningfull, e.g. `someDescriptiveNameLine`.